<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionPropertyTemplateEditorDialog</class>
 <widget class="QDialog" name="ActionPropertyTemplateEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="text">
      <string>&amp;Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="6">
    <widget class="QLineEdit" name="edtCode"/>
   </item>
   <item row="0" column="7">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>321</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblFederalCode">
     <property name="text">
      <string>&amp;Федеральный код</string>
     </property>
     <property name="buddy">
      <cstring>edtFederalCode</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="6">
    <widget class="QLineEdit" name="edtFederalCode"/>
   </item>
   <item row="1" column="7">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>171</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblRegionalCode">
     <property name="text">
      <string>&amp;Региональный код</string>
     </property>
     <property name="buddy">
      <cstring>edtRegionalCode</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="6">
    <widget class="QLineEdit" name="edtRegionalCode"/>
   </item>
   <item row="2" column="7">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>171</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblGroup">
     <property name="text">
      <string>&amp;Группа</string>
     </property>
     <property name="buddy">
      <cstring>cmbGroup</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="7">
    <widget class="CActionPropertyTemplateComboBox" name="cmbGroup"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>На&amp;именование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="7">
    <widget class="QLineEdit" name="edtName">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblAbbrev">
     <property name="text">
      <string>&amp;Сокращение</string>
     </property>
     <property name="buddy">
      <cstring>edtAbbrev</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="7">
    <widget class="QLineEdit" name="edtAbbrev">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblSex">
     <property name="text">
      <string>&amp;Пол</string>
     </property>
     <property name="buddy">
      <cstring>cmbSex</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="cmbSex">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ж</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="2" colspan="6">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>231</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblAge">
     <property name="text">
      <string>&amp;Возраст</string>
     </property>
     <property name="buddy">
      <cstring>cmbBegAgeUnit</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="cmbBegAgeUnit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Д</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Н</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Г</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="edtBegAgeCount">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMask">
      <string>000; </string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="lblAgeSep">
     <property name="text">
      <string>по</string>
     </property>
     <property name="buddy">
      <cstring>cmbEndAgeUnit</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="4" colspan="2">
    <widget class="QComboBox" name="cmbEndAgeUnit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Д</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Н</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Г</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="6">
    <widget class="QLineEdit" name="edtEndAgeCount">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMask">
      <string>000; </string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="7" column="7">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>71</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblService">
     <property name="text">
      <string>&amp;Услуга</string>
     </property>
     <property name="buddy">
      <cstring>cmbService</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="7">
    <widget class="CRBComboBox" name="cmbService"/>
   </item>
   <item row="9" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>73</width>
       <height>231</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0" colspan="8">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CActionPropertyTemplateComboBox</class>
   <extends>QComboBox</extends>
   <header>Events/ActionPropertyTemplateComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtCode</tabstop>
  <tabstop>edtFederalCode</tabstop>
  <tabstop>edtRegionalCode</tabstop>
  <tabstop>cmbGroup</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtAbbrev</tabstop>
  <tabstop>cmbSex</tabstop>
  <tabstop>cmbBegAgeUnit</tabstop>
  <tabstop>edtBegAgeCount</tabstop>
  <tabstop>cmbEndAgeUnit</tabstop>
  <tabstop>edtEndAgeCount</tabstop>
  <tabstop>cmbService</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ActionPropertyTemplateEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ActionPropertyTemplateEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
